Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -763,6 +767,16 @@ static inline bool computeIsAlwaysOnLoggingAllowed(NetworkSession& session)
     m_task.get().priority = toNSURLSessionTaskPriority(priority);
 }
 
+#if ENABLE(INSPECTOR_NETWORK_THROTTLING)
+
+void NetworkDataTaskCocoa::setEmulatedConditions(const std::optional<int64_t>& bytesPerSecondLimit)
+{
+    if (bytesPerSecondLimit)
```

Comment:
I wonder if somebody should assert this is positive... I never liked `int64_t`.

Reply:
Is there a reason we aren't using uint64_t instead? Seems like we are wasting half of `int64_t`'s potential since it can't be lower than 0.

Reply:
CFNetwork uses `int64_t` for their SPI